api/config/config_preprod.json

```diff
@@ -93,10 +93,8 @@
   "absorb": {
     "externalDepartmentId": "Researchers",
     "enabledForNewUsers": false,
-    "rtTrainingCourseId": "TODO",
-    "ctTrainingCourseId": "TODO",
-    "samlIdentityProviderId": "TODO",
-    "samlServiceProviderId": "TODO"
+    "samlIdentityProviderId": "C02celxus",
+    "samlServiceProviderId": "638082479253"
```

Comment:
Obtained from the URL used in the "End-to-end test" section of the [SSO configuration doc](https://docs.google.com/document/d/1xgcvow0xPL6K4vxyM3PVlW-1E5Ye3X1Y5wZSD5rrPlc/edit?resourcekey=0-q_6-vFevph2piD88EfX-1g#bookmark=id.wesss7shr9s3).